kubetest/kops.go

```diff
@@ -261,14 +291,20 @@ func (k kops) Up() error {
 		}
 	}
 
-	var featureFlags []string
+	featureFlags := []string{"SpecOverrideFlag"}
 
 	createArgs := []string{
 		"create", "cluster",
 		"--name", k.cluster,
 		"--ssh-public-key", k.sshPublicKey,
 		"--node-count", strconv.Itoa(k.nodes),
+		"--master-size", "c4.large",
 		"--zones", strings.Join(k.zones, ","),
+		"--override", "'cluster.spec.etcdClusters[*].version=3.1.10'",
```

Comment:
are we pinning etcd version for all the kops job now? (does etcd 3.1 work with older releases?)

Reply:
good point, what versions are we testing.